tests/test_primitives/test_exceptions/test_pickle_unwrap_failed_error.py

```diff
@@ -7,31 +7,42 @@
 
 def test_pickle_unwrap_failed_error_from_maybe():
     """Ensures that UnwrapFailedError with Maybe can be pickled."""
+    error = None
+    serialized = None
+    error = None
```

Comment:
```suggestion
    serialized = None
```

We only need this var :)